build.sbt

```diff
@@ -61,20 +63,22 @@ dynverSeparator in ThisBuild := "-"
 val bouncycastleBcprov = "org.bouncycastle" % "bcprov-jdk15on" % "1.64"
 val scrypto            = "org.scorexfoundation" %% "scrypto" % "2.1.10"
 val scorexUtil         = "org.scorexfoundation" %% "scorex-util" % "0.1.8"
-val debox              = "org.spire-math" %% "debox" % "0.8.0"
-val kiama              = "org.bitbucket.inkytonik.kiama" %% "kiama" % "2.1.0"
-val fastparse          = "com.lihaoyi" %% "fastparse" % "1.0.0"
+val debox              = "org.scorexfoundation" %% "debox" % "0.9.0"
```

Comment:
Not reviewed internal (also, not describing changes from parent library etc)

Reply:
the original debox project was abandoned. The PRs there are not reviewed for years.